docs/graph.md

```diff
@@ -190,6 +190,17 @@ stateDiagram-v2
   Increment --> DivisibleBy5
 ```
 
+!!! tips
+    In order to visualize a graph within a `jupyter-notebook`, `IPython.display` needs to be used
+    ```python
+    from IPython.display import Image, display
+    display(
+        Image(
+            fives_graph.mermaid_image(start_node=DivisibleBy5)
+            )
+    )
```

Comment:
you'll need to format this code correctly, you should be able to run `uv run pytest tests/test_examples.py --update-examples` to do it automaticaly.